plugins/modules/database/misc/redis.py

```diff
@@ -66,13 +70,15 @@
         type: str
     name:
         description:
-            - A redis config key.
+            - A redis config or db key.
         type: str
+        aliases: [ key ]
     value:
         description:
             - A redis config value. When memory size is needed, it is possible
               to specify it in the usal form of 1KB, 2M, 400MB where the base is 1024.
               Units are case insensitive i.e. 1m = 1mb = 1M = 1MB.
+            - Or a redis db value
```

Comment:
What is a "db value" (do you mean "DB value", with DB == database?), and how is it different from a "config value"?